llvm/lib/Target/RISCV/MCTargetDesc/RISCVTargetStreamer.cpp

```diff
@@ -14,12 +14,20 @@
 #include "RISCVBaseInfo.h"
 #include "RISCVMCTargetDesc.h"
 #include "llvm/MC/MCSymbol.h"
+#include "llvm/Support/CommandLine.h"
 #include "llvm/Support/FormattedStream.h"
 #include "llvm/Support/RISCVAttributes.h"
 #include "llvm/TargetParser/RISCVISAInfo.h"
 
 using namespace llvm;
 
+// This option controls wether or not we emit ELF attributes for ABI features,
+// like RISC-V atomics or X3 usage.
+static cl::opt<bool> RiscvAbiAttr(
+    "riscv-abi-attributes",
+    cl::desc("Enable emitting RISC-V ELF attributes for ABI features"),
+    cl::init(false), cl::Hidden);
```

Comment:
remove `cl::init(false),` the convention is to omit default false.

Reply:
ugh, I forgot to fix that, fixed up a bad merge. Thanks for noticing that.